src/components/sideNav.js

```diff
@@ -48,34 +139,65 @@ export default function sideNav(props) {
       `}
       render={data => {
         const { Date, OfficeElections } = data.allElection.edges[0].node
-        const menu = formatMenu(OfficeElections)
+        const [menu, options] = formatMenu(OfficeElections)
         const sections = Object.keys(menu)
+        let url = window.location.href.split("/")
+        url = url[url.length - 1]
+          .split("-")
+          .map(word => word[0].toUpperCase() + word.slice(1))
+          .join(" ")
```

Comment:
what is this getting? the office title? maybe `selectedTitle` would be a better name

Reply:
yes, it's getting the office title so that I can highlight the current menu item. It's definitely hacky; I figured it'd be okay because we have total control over the URL, but I'm going to refactor it to something a bit more robust.